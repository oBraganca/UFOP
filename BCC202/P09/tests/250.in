250
Luana 38678 7.2
AnaCecilia 594812 4.2
JoaoGuilherme 947894 3.5
Mirella 772217 0.8
Bruno 466009 7.4
Sophia 204892 9.3
Vinicius 477856 4.9
CarlosEduardo 97537 4.1
AnaLivia 958573 3.8
AnaJulia 225714 7.5
Isabella 708605 1.8
Alice 591295 9.8
Nicolas 810469 6.7
Valentina 568068 0.8
Emanuelly 464132 9.2
Henry 767980 6.1
Pietro 206236 6.5
Nicole 412071 6.6
Eduarda 148561 5.3
Anthony 884407 7.0
Henrique 912627 9.9
Theo 601232 5.6
Bento 48402 8.3
Marina 111383 6.6
Lorenzo 112915 2.3
MariaFernanda 444213 8.2
JoaoGabriel 706497 7.9
LuizFelipe 57289 8.5
Raul 846591 7.9
Allana 112557 1.7
Tomas 182463 3.2
Lavinia 905255 1.1
MariaClara 466639 5.5
MariaSophia 859954 9.1
Danilo 283128 2.8
PedroLucas 512106 3.5
Antonella 556508 2.8
Bianca 361366 5.1
Isadora 322027 6.9
Larissa 541277 5.8
MariaCecilia 934443 3.1
DaviLucas 78465 7.6
Kaique 658066 5.4
Miguel 716805 9.8
Milena 862322 7.6
LuizHenrique 230565 7.5
Lorena 769408 8.2
Fernando 793261 9.8
Agatha 520265 9.0
Joao 421727 3.3
Bernardo 568911 9.0
Esther 611467 2.0
JoaoPedro 654966 7.6
AnaSophia 965709 3.5
Arthur 915870 6.2
Lara 587773 6.0
Pietra 411427 1.0
Emanuel 330019 8.6
Miguel 849282 1.1
Caio 203168 0.8
Heloise 495122 5.3
Oliver 606917 5.2
ArthurHenrique 419950 4.4
LuizMiguel 503338 8.2
Erick 49585 3.6
Eduardo 88070 0.5
MariaLuiza 546679 0.6
LucasGabriel 338604 4.5
Vicente 922826 3.0
ArthurGabriel 911367 4.7
Rafael 728319 4.2
Mariana 925524 7.6
Lucas 306725 3.9
Otavio 176655 1.3
Joaquim 310478 0.7
DaviLucca 46497 1.5
EnzoMiguel 68156 2.7
Gael 650140 8.7
Daniel 95650 5.7
Thiago 555349 7.1
Lucca 985456 8.7
Clara 604441 2.3
AnaLaura 827166 6.1
Caleb 761845 7.8
Breno 35368 0.8
Leonardo 468338 9.5
Benicio 512093 8.7
AnaClara 986334 4.5
Lais 796337 7.0
Clarice 940768 2.2
Analu 27006 6.4
LuizOtavio 694090 2.5
Yuri 444411 2.3
Thomas 290817 3.7
Murilo 490026 6.9
DaviLuiz 384999 2.1
DaviMiguel 933667 4.7
ArthurMiguel 896720 4.9
Isabelly 213644 3.4
MariaIsis 372978 7.4
Aurora 740907 1.4
Laura 706418 6.6
Alicia 308542 7.7
MariaEduarda 192734 9.8
Elisa 53013 8.2
Gustavo 69985 8.7
Francisco 496829 5.3
Luiza 402724 3.8
Livia 822214 7.8
Fernanda 687436 2.5
Augusto 519292 4.9
Bruna 55572 7.9
Rebeca 38441 3.4
Joana 504236 1.7
Enrico 642735 8.1
Sophie 322293 1.0
Maria 189263 6.5
Amanda 51030 1.3
Cecilia 823309 9.9
EnzoGabriel 944836 6.8
Liz 171235 2.7
Jose 995071 7.7
Davi 167724 4.5
LuizGustavo 120668 6.7
Catarina 453624 3.3
Giovanna 239990 3.3
PedroHenrique 936991 7.0
MariaValentina 381471 1.2
Gabriela 705957 1.1
Yago 342755 2.8
Levi 5624 1.5
Yasmin 893510 1.0
AnaLuiza 985589 8.1
Mariah 32975 2.2
Ian 170080 5.0
Felipe 776261 9.1
Gabriel 242549 2.8
Heitor 782164 7.2
Maya 543804 6.6
Leticia 208851 1.1
Gabriel 929490 2.6
Guilherme 788940 5.2
MariaAlice 946442 5.0
JoaoMiguel 367896 1.9
Samuel 736537 4.3
MariaHelena 891204 8.8
VictorHugo 870171 8.1
Sarah 655088 4.6
Manuela 242006 4.1
Ryan 411278 5.3
JoaoLucas 956528 2.6
Eloa 930839 6.6
Matheus 904727 4.2
Henrique 624606 1.5
Noah 789400 7.1
Gabrielly 637289 0.8
Rodrigo 232401 7.1
Anthony 884034 4.9
MariaFlor 615804 8.5
Benjamin 652016 1.2
PedroMiguel 333927 2.2
Caua 808701 3.3
MariaLaura 603596 0.8
Nathan 82462 4.8
Victor 282272 7.8
Vitoria 626409 8.9
Ruan 841431 8.8
Malu 163354 6.1
MariaJulia 144795 2.5
Kauê 907333 2.5
Rafaela 361388 2.3
AnaLiz 217857 4.9
Isaac 916363 1.8
Antonia 486568 2.7
Julia 321338 8.1
Isis 439594 3.4
AnaBeatriz 893092 9.0
JoaoVictor 372612 3.8
Luna 278434 0.9
Pedro 955306 0.9
Bryan 287929 6.2
Ana 84195 5.4
Enzo 481729 7.4
Ayla 689256 1.4
Emilly 856178 5.3
Carolina 294218 5.6
Louise 952108 2.8
Maitê 773992 4.7
Stella 408772 3.9
Melissa 969453 4.7
Andre 850158 8.7
Antonio 773336 9.1
Helena 465983 4.8
Isabel 132034 4.4
AnaVitoria 434588 4.5
Heloisa 197008 7.6
Olivia 292542 2.6
MariaVitoria 117936 3.8
Martin 763144 7.0
Beatriz 101060 9.7
MariaHeloisa 41390 5.9
Luan 249233 5.3
Luana 448405 6.6
AnaCecilia 672064 7.8
JoaoGuilherme 238497 2.3
Mirella 658023 1.4
Bruno 466955 8.5
Sophia 211893 1.6
Vinicius 743518 8.3
CarlosEduardo 352968 1.6
AnaLivia 293019 1.9
AnaJulia 876159 4.7
Isabella 684698 9.1
Alice 147072 2.3
Nicolas 386916 1.0
Valentina 481156 4.2
Emanuelly 218430 4.4
Henry 918092 9.6
Pietro 690583 6.5
Nicole 622871 6.5
Eduarda 829800 1.7
Anthony 713748 4.2
Henrique 570863 8.0
Theo 437342 2.3
Bento 310131 5.3
Marina 280147 1.8
Lorenzo 71055 3.4
MariaFernanda 432199 6.0
JoaoGabriel 510130 8.9
LuizFelipe 514493 3.5
Raul 548723 9.5
Allana 645845 3.6
Tomas 401369 1.3
Lavinia 265678 4.8
MariaClara 23907 4.7
MariaSophia 982564 9.6
Danilo 987110 7.3
PedroLucas 138704 8.7
Antonella 665199 0.9
Bianca 536524 3.5
Isadora 58493 8.0
Larissa 672354 8.7
MariaCecilia 309327 6.8
DaviLucas 227317 1.0
Kaique 123523 4.4
Miguel 712083 7.1
Milena 171128 8.5
LuizHenrique 282174 1.7
Lorena 323058 9.7
Fernando 172941 5.9
