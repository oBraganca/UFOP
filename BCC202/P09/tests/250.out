Eduardo 88070 0.5
MariaLuiza 546679 0.6
Joaquim 310478 0.7
Breno 35368 0.8
Caio 203168 0.8
Valentina 568068 0.8
MariaLaura 603596 0.8
Gabrielly 637289 0.8
Mirella 772217 0.8
Luna 278434 0.9
Antonella 665199 0.9
Pedro 955306 0.9
DaviLucas 227317 1.0
Sophie 322293 1.0
Nicolas 386916 1.0
Pietra 411427 1.0
Yasmin 893510 1.0
Leticia 208851 1.1
Gabriela 705957 1.1
Miguel 849282 1.1
Lavinia 905255 1.1
MariaValentina 381471 1.2
Benjamin 652016 1.2
Amanda 51030 1.3
Otavio 176655 1.3
Tomas 401369 1.3
Mirella 658023 1.4
Ayla 689256 1.4
Aurora 740907 1.4
Levi 5624 1.5
DaviLucca 46497 1.5
Henrique 624606 1.5
Sophia 211893 1.6
CarlosEduardo 352968 1.6
Allana 112557 1.7
LuizHenrique 282174 1.7
Joana 504236 1.7
Eduarda 829800 1.7
Marina 280147 1.8
Isabella 708605 1.8
Isaac 916363 1.8
AnaLivia 293019 1.9
JoaoMiguel 367896 1.9
Esther 611467 2.0
DaviLuiz 384999 2.1
Mariah 32975 2.2
PedroMiguel 333927 2.2
Clarice 940768 2.2
Lorenzo 112915 2.3
Alice 147072 2.3
JoaoGuilherme 238497 2.3
Rafaela 361388 2.3
Theo 437342 2.3
Yuri 444411 2.3
Clara 604441 2.3
MariaJulia 144795 2.5
Fernanda 687436 2.5
LuizOtavio 694090 2.5
Kauê 907333 2.5
Olivia 292542 2.6
Gabriel 929490 2.6
JoaoLucas 956528 2.6
EnzoMiguel 68156 2.7
Liz 171235 2.7
Antonia 486568 2.7
Gabriel 242549 2.8
Danilo 283128 2.8
Yago 342755 2.8
Antonella 556508 2.8
Louise 952108 2.8
Vicente 922826 3.0
MariaCecilia 934443 3.1
Tomas 182463 3.2
Giovanna 239990 3.3
Joao 421727 3.3
Catarina 453624 3.3
Caua 808701 3.3
Rebeca 38441 3.4
Lorenzo 71055 3.4
Isabelly 213644 3.4
Isis 439594 3.4
PedroLucas 512106 3.5
LuizFelipe 514493 3.5
Bianca 536524 3.5
JoaoGuilherme 947894 3.5
AnaSophia 965709 3.5
Erick 49585 3.6
Allana 645845 3.6
Thomas 290817 3.7
MariaVitoria 117936 3.8
JoaoVictor 372612 3.8
Luiza 402724 3.8
AnaLivia 958573 3.8
Lucas 306725 3.9
Stella 408772 3.9
CarlosEduardo 97537 4.1
Manuela 242006 4.1
Valentina 481156 4.2
AnaCecilia 594812 4.2
Anthony 713748 4.2
Rafael 728319 4.2
Matheus 904727 4.2
Samuel 736537 4.3
Kaique 123523 4.4
Isabel 132034 4.4
Emanuelly 218430 4.4
ArthurHenrique 419950 4.4
Davi 167724 4.5
LucasGabriel 338604 4.5
AnaVitoria 434588 4.5
AnaClara 986334 4.5
Sarah 655088 4.6
MariaClara 23907 4.7
Maitê 773992 4.7
AnaJulia 876159 4.7
ArthurGabriel 911367 4.7
DaviMiguel 933667 4.7
Melissa 969453 4.7
Nathan 82462 4.8
Lavinia 265678 4.8
Helena 465983 4.8
AnaLiz 217857 4.9
Vinicius 477856 4.9
Augusto 519292 4.9
Anthony 884034 4.9
ArthurMiguel 896720 4.9
Ian 170080 5.0
MariaAlice 946442 5.0
Bianca 361366 5.1
Oliver 606917 5.2
Guilherme 788940 5.2
Eduarda 148561 5.3
Luan 249233 5.3
Bento 310131 5.3
Ryan 411278 5.3
Heloise 495122 5.3
Francisco 496829 5.3
Emilly 856178 5.3
Ana 84195 5.4
Kaique 658066 5.4
MariaClara 466639 5.5
Carolina 294218 5.6
Theo 601232 5.6
Daniel 95650 5.7
Larissa 541277 5.8
MariaHeloisa 41390 5.9
Fernando 172941 5.9
MariaFernanda 432199 6.0
Lara 587773 6.0
Malu 163354 6.1
Henry 767980 6.1
AnaLaura 827166 6.1
Bryan 287929 6.2
Arthur 915870 6.2
Analu 27006 6.4
Maria 189263 6.5
Pietro 206236 6.5
Nicole 622871 6.5
Pietro 690583 6.5
Marina 111383 6.6
Nicole 412071 6.6
Luana 448405 6.6
Maya 543804 6.6
Laura 706418 6.6
Eloa 930839 6.6
LuizGustavo 120668 6.7
Nicolas 810469 6.7
MariaCecilia 309327 6.8
EnzoGabriel 944836 6.8
Isadora 322027 6.9
Murilo 490026 6.9
Martin 763144 7.0
Lais 796337 7.0
Anthony 884407 7.0
PedroHenrique 936991 7.0
Rodrigo 232401 7.1
Thiago 555349 7.1
Miguel 712083 7.1
Noah 789400 7.1
Luana 38678 7.2
Heitor 782164 7.2
Danilo 987110 7.3
MariaIsis 372978 7.4
Bruno 466009 7.4
Enzo 481729 7.4
AnaJulia 225714 7.5
LuizHenrique 230565 7.5
DaviLucas 78465 7.6
Heloisa 197008 7.6
JoaoPedro 654966 7.6
Milena 862322 7.6
Mariana 925524 7.6
Alicia 308542 7.7
Jose 995071 7.7
Victor 282272 7.8
AnaCecilia 672064 7.8
Caleb 761845 7.8
Livia 822214 7.8
Bruna 55572 7.9
JoaoGabriel 706497 7.9
Raul 846591 7.9
Isadora 58493 8.0
Henrique 570863 8.0
Julia 321338 8.1
Enrico 642735 8.1
VictorHugo 870171 8.1
AnaLuiza 985589 8.1
Elisa 53013 8.2
MariaFernanda 444213 8.2
LuizMiguel 503338 8.2
Lorena 769408 8.2
Bento 48402 8.3
Vinicius 743518 8.3
LuizFelipe 57289 8.5
Milena 171128 8.5
Bruno 466955 8.5
MariaFlor 615804 8.5
Emanuel 330019 8.6
Gustavo 69985 8.7
PedroLucas 138704 8.7
Benicio 512093 8.7
Gael 650140 8.7
Larissa 672354 8.7
Andre 850158 8.7
Lucca 985456 8.7
Ruan 841431 8.8
MariaHelena 891204 8.8
JoaoGabriel 510130 8.9
Vitoria 626409 8.9
Agatha 520265 9.0
Bernardo 568911 9.0
AnaBeatriz 893092 9.0
Isabella 684698 9.1
Antonio 773336 9.1
Felipe 776261 9.1
MariaSophia 859954 9.1
Emanuelly 464132 9.2
Sophia 204892 9.3
Leonardo 468338 9.5
Raul 548723 9.5
Henry 918092 9.6
MariaSophia 982564 9.6
Beatriz 101060 9.7
Lorena 323058 9.7
MariaEduarda 192734 9.8
Alice 591295 9.8
Miguel 716805 9.8
Fernando 793261 9.8
Cecilia 823309 9.9
Henrique 912627 9.9
